doc/releases/changelog-dev.md

```diff
@@ -173,6 +173,17 @@
   [noise channels](https://docs.pennylane.ai/en/latest/introduction/operations.html#noisy-channels)
   when operating in finite-shot mode.
 
+* A function called `apply_operation` has been added to the new `qutrit_mixed` module found in `qml.devices` that applies operations to device-compatible states.
+  [(#5032)](https://github.com/PennyLaneAI/pennylane/pull/5032)
```

Comment:
This should not be here?

Reply:
Looks like a merge conflict.

Reply:
Not sure what to do here. This came from the merge. Should I remove it?